### PR TITLE
Basic Cassandra backend code

### PR DESCRIPTION
**Pull request by krishnan-chandra**
Constructor for the backend has been created, now to make the logic for queues.

### CODE REVIEW

File: CassandraBackend.js [hunk 1]
```diff
@@ -1,8 +1,57 @@
...
+  cass = require('node-cassandra-cql'),
+  consistencies = cass.types.consistencies,
+  uuid = require('node-uuid'),
+  async = require('async');
+
 // Constructor
-function CassandraBackend(config) {
-	this.config = config;
+function CassandraBackend(name, config, callback) {
+  events.EventEmitter.call(this);
```
Comment (gwicke) [inline CassandraBackend.js]: Do we actually need to subclass EventEmitter? I think we can get away without one.

File: CassandraBackend.js [hunk 2]
```diff
@@ -1,8 +1,57 @@
...
+  //  console.log('log event: %s -- %j', level, message);
+  //});
+  callback();
 }
 
+// CassandraBackend.prototype._loadCommits = function(cb) {
+//     this.client.execute("SELECT * from tests", null, this.consistencies.read, queryCB);
+// }
+
+util.inherits(CassandraBackend, events.EventEmitter);
```
Comment (gwicke) [inline CassandraBackend.js]: Might not need this.

**Comment by gwicke** [comment]
A good first step! I'm not sure that we need the EventEmitter, see inline comments.

File: package.json [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
-		"mysql": "2.x.x",
-		"mysql-queues": "1.x.x",
-		"express": "2.5.x",
-		"handlebars": "~1.3.0"
-	},
-	"main": "server.js"
+  "name": "mediawiki-parsoid-testserv",
+  "description": "Mediawiki parser for the VisualEditor - test server.",
+  "version": "0.0.1",
+  "dependencies": {
```
Comment (shinnoki) [inline package.json:5]: need cassandra-cql dependencies.
"node-cassandra-cql": "git+https://github.com/gwicke/node-cassandra-cql#master"

**Pull request merged**